src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.cpp

```diff
@@ -1269,11 +1269,9 @@ void VTKPolyDataWriterInterface::write( real64 const time,
                                         integer const cycle,
                                         DomainPartition const & domain )
 {
-  // This guard prevents crashes observed on MacOS due to a floating point exception
+  // This guard prevents crashes due to a floating point exception (SIGFPE)
```

Comment:
doesn't this suppress a FPE that we might not want suppressed?

Reply:
Once we update VTK to 9.3 (first RC is already out) we won't be needing these workarounds, since we [upstreamed a fix](https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10202) a while ago. Until then it's ok to have the suppression IMO.